File: PYTHON/Kode Program/Processor.py
from Komponen import Komponen

class Processor(Komponen):
    def __init__(self, core_count=0, clock_speed=0.0, merk="", harga=0):
        super().__init__(merk, harga)
        self._core_count = core_count
        self._clock_speed = clock_speed

    def set_core_count(self, core_count):
        self._core_count = core_count

    def get_core_count(self):
        return self._core_count

    def set_clock_speed(self, clock_speed):
        self._clock_speed = clock_speed

    def get_clock_speed(self):
        return self._clock_speed

File: JAVA/Kode Program/Processor.java
public class Processor extends Komponen {
    private int coreCount;
    private double clockSpeed;

    public Processor(int coreCount, double clockSpeed, String merk, int harga) {
        super(merk, harga);
        this.coreCount = coreCount;
        this.clockSpeed = clockSpeed;
    }

    public void setCoreCount(int coreCount) {
        this.coreCount = coreCount;
    }

    public int getCoreCount() {
        return this.coreCount;
    }

    public void setClockSpeed(double clockSpeed) {
        this.clockSpeed = clockSpeed;
    }

    public double getClockSpeed() {
        return this.clockSpeed;
    }
}
